app/assets/javascripts/controllers/users-controller.coffee

```diff
@@ -0,0 +1,32 @@
+UsersController = ($scope, $auth, $state, Flash) ->
+  ctrl = @
+  ctrl.registrationForm = {}
+  
+  ctrl.create = (form) ->
+    if form.$valid
+      $auth.submitRegistration({
+        username: form.username.$modelValue,
+        password: form.password.$modelValue,
+        password_confirmation: form.password_confirmation.$modelValue
+      }).then ( (response) ->
+        Flash.create 'success', "<b>Well done!</b> You’re successfully registered!"
```

Comment:
i18n?

Reply:
not need there

Reply:
no any word about localization in technical task

Reply:
I done it by user stories in tech task.
It's stupid change only one flash label in app.

Reply:
resolved

Reply:
@kirillshevch Don't forget apply pull request.

Reply:
@kirillshevch translation added.

Reply:
@kirillshevch i added it

Reply:
Please contact to HR
@edlvj